extensions/lifecycle/tasks/LifecycleUpdateTransitionTask.js

```diff
@@ -98,6 +98,19 @@ class LifecycleUpdateTransitionTask extends BackbeatTask {
         this.gcProducer.publishActionEntry(gcEntry, done);
     }
 
+    _wasObjectModified(entry, objMD, log) {
+        const { lastModified } = entry.getAttribute('target');
+        const objectWasModified = lastModified !== objMD.getLastModified();
+        if (objectWasModified) {
+            log.info('object LastModified date changed during lifecycle ' +
+                     'transition processing');
+            Object.assign({
```

Comment:
This should be passed as second argument of `log.info`

Reply:
Thanks for that. Nice catch.